### DISCUSSION TITLE
tailwind CSS position  sticky top no work

### DISCUSSION THREAD

**Discussion by sujon-258549** [tailwindlabs/tailwindcss#14953] [Help]
tailwind CSS position  sticky top no work

**Comment by wongjn** [comment]
Consider checking if it works with regular CSS, if it doesn't, then consider reviewing how `position: sticky` works.

Otherwise, consider checking if the appropriate CSS is being applied to the element if not, then consider checking:
- Tailwind has compiled since you added the classes.
- The `content` file globs sufficiently cover the file that the code appears in.
- [String concatenation/interpolation is not used to build the class name](https://tailwindcss.com/docs/content-configuration#dynamic-class-names). 
- The class names would resolve to a Tailwind class that Tailwind would generate a rule for.

Otherwise, consider providing a project that reproduces the unexpected behavior.
